ConsoleApp2/ConsoleApp2.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Was this file intended to be in this PR?

Reply:
Removed